### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by bartko-s**
Slovak translation update

### CODE REVIEW

**Comment by bartko-s** [comment]
ping @Maks3w

**Comment by bartko-s** [comment]
Can someone @weierophinney , @Maks3w merge this pull request? And also close #12 pull request because this pull request and #12 pull request are almost identical. Thank you.

**Comment by mmoll** [comment]
@bartko-s could you send the PR (if possible as one single commit comtaining the changes) to the master branch (per #26)? If it's too much of a hassle, I can also open a new PR with the respective commits cherry-picked.

**Comment by bartko-s** [comment]
I don't know what exactly you want from me. This PR has no conflict with develop branch.

**Comment by mmoll** [comment]
@bartko-s the PR is from your slovak-translation-update branch to the master branch of this repo, but it includes all the commits from the develop branch, too, as far as I can see.

**Comment by bartko-s** [comment]
should be better now

**Comment by mmoll** [comment]
@inecas If you have the time, could you review this as a native speaker? :)
As a non-native speaker, but understanding Slovak, I'm 👍 this.

File: languages/sk/Zend_Validate.php [hunk 1]
```diff
@@ -8,10 +8,16 @@
...
- * SK-Revision: 30.Sep.2013
+ * SK-Revision: 3.Dec.2016
  */
 return [
-    // Zend\I18n\Validate\Alnum
+    // Zend\Authentication\Validator\Authentication
+    "Invalid identity" => "Neplatná identita",
+    "Identity is ambiguous" => "Identita je nejednoznačná",
+    "Invalid password" => "Neplatné heslo",
+    "Authentication failed" => "Prihlasenie zlyhalo",
```
Comment (iNecas) [inline languages/sk/Zend_Validate.php]: Prihlásenie

File: languages/sk/Zend_Validate.php [hunk 2]
```diff
@@ -8,10 +8,16 @@
...
 
 /**
- * SK-Revision: 30.Sep.2013
+ * SK-Revision: 3.Dec.2016
  */
 return [
-    // Zend\I18n\Validate\Alnum
+    // Zend\Authentication\Validator\Authentication
+    "Invalid identity" => "Neplatná identita",
+    "Identity is ambiguous" => "Identita je nejednoznačná",
```
Comment (iNecas) [inline languages/sk/Zend_Validate.php]: `Identita nie je jednoznačná` would sound more natural, to me

**Review by iNecas — APPROVED** [review]

**Comment by iNecas** [comment]
One suggestion and one missing `á`: I haven't found anything else

**Comment by froschdesign** [comment]
@bartko-s 
Can you confirm the corrections from @iNecas and update your PR?

Thanks!

**Comment by froschdesign** [comment]
@bartko-s 
Please update also the comments with validator classes:

- `Zend\I18n\Validator\Float` to `Zend\I18n\Validator\IsFloat`
- `Zend\I18n\Validator\Int` to `Zend\I18n\Validator\IsInt`

That would be great. Thanks!

**Comment by bartko-s** [comment]
All suggested corrections was done.

**Review by iNecas — APPROVED** [review]

**Comment by weierophinney** [comment]
Thanks, @bartko-s! Merged to develop for release with 2.6.0.

**Comment by bartko-s** [comment]
Great. It took only 27 months.

**Comment by Ocramius** [comment]
It took only 0 moneys too ;-)

**Pull request closed without merging**